app/src/test/java/org/mozilla/fenix/settings/logins/LoginsListViewHolderTest.kt

```diff
@@ -51,6 +51,17 @@ class LoginsListViewHolderTest {
 
         assertEquals("mozilla.org", binding.webAddressView.text)
         assertEquals("admin", binding.usernameView.text)
+
+        // test for proper simplification of mobile URLs
+        val mLogin = baseLogin.copy(origin = "https://m.mozilla.org")
+        holder.bind(mLogin)
+
+        assertEquals("mozilla.org", binding.webAddressView.text)
+
+        val mobileLogin = baseLogin.copy(origin = "https://mobile.mozilla.org")
+        holder.bind(mobileLogin)
+
+        assertEquals("mozilla.org", binding.webAddressView.text)
```

Comment:
Could you add this as a separate @Test?

Reply:
Done. I also factored out the holder definition.